specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/inputs.json

```diff
@@ -660,6 +676,35 @@
         }
       }
     },
+    "RawReferenceInputDataSource": {
+      "description": "Describes a raw input data source. This data source type is only applicable/usable when using the query testing API. You cannot create a job with this data source type or add an input/output of this data source type to an existing job.",
+      "x-ms-discriminator-value": "Raw",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ReferenceInputDataSource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RawInputDatasourceProperties",
+          "description": "The properties that are associated with a raw input containing stream data.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RawInputDatasourceProperties": {
+      "description": "The properties that are associated with a raw input.",
+      "properties": {
+        "payload": {
+          "description": "The JSON serialized content of the input data. ",
```

Comment:
The content does not have to be serialized specifically to JSON right? It is just serialized content from what I understand